From d1412de35200dddc6ff80807f08b7ef5f9159c82 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Thu, 20 Aug 2015 11:41:53 +0900
Subject: [PATCH 05/85] arm64: dts: r8a7795: Fix timer frequency

Currently CNTFRQ register is set 16.666666 MHz by firmware.
This register has to be set 1/2 of EXTAL.
But current R-Car H3 board (Salvator-X) has 16.666666 MHz
crystal oscillator instead of 33.333333 MHz.
Therefore this register should be set 8.333333 MHz.

This patch fixes CNTFRQ register value by setting clock-frequency
property of timer node to 8.333333 MHz.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index 0940549..89f2467 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -88,6 +88,7 @@
 					(GIC_CPU_MASK_SIMPLE(4) | IRQ_TYPE_LEVEL_LOW)>,
 				     <GIC_PPI 10
 					(GIC_CPU_MASK_SIMPLE(4) | IRQ_TYPE_LEVEL_LOW)>;
+			clock-frequency = <8333333>;
 		};
 
 		clocks {
-- 
1.7.9.5

